package.json

```diff
@@ -33,7 +34,10 @@
         "@react-aria/overlays": "^3.8.1",
         "@react-stately/combobox": "^3.0.4",
         "dash": "^3.24.21",
+        "moment": "^2.29.4",

```

Comment:
Please avoid moment choice something like date-fn. Moment not really support and is a bloat